pint/compat/__init__.py

```diff
@@ -66,6 +67,26 @@ def u(x):
 except ImportError:
     from itertools import izip_longest as zip_longest
 
+# TODO: remove this warning after v0.10
+class BehaviorChangeWarning(UserWarning):
+    pass
+_msg = ('The way pint handles numpy operations has changed with '
+'the implementation of NEP 18. Unimplemented numpy operations '
+'will now fail instead of making assumptions about units. Some '
+'functions, eg concat, will now return Quanties with units, '
+'where they returned ndarrays previously. See '
+'https://github.com/hgrecco/pint/pull/xxxx. '
+'To hide this warning use the following code to import pint:'
+"""
+import warnings
+with warnings.catch_warnings():
+    warnings.simplefilter("ignore")
+    import pint
+To disable the new behavior, see
+https://www.numpy.org/neps/nep-0018-array-function-protocol.html#implementation
+---
+""")
```

Comment:
Wouldn't it be nicer to have the whole thing wrapped by ``"""``? (plus strip/rstrip/dedent as needed)